e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js

```diff
@@ -578,35 +578,7 @@ describe("scenarios > visualizations > line chart", () => {
     });
   });
 
-  it(
```

Comment:
I removes this test as it duplicates the one from `chart_drill` (I updated the name to `name: "Brush Date Temporal Filter",` )